fullstop-plugins/fullstop-registry-plugin/src/main/java/org/zalando/stups/fullstop/plugin/config/RegistryPluginProperties.java

```diff
@@ -49,15 +52,14 @@
         return defaultApprovals;
     }
 
-    public Set<String> getDefaultApprovals() {
-        return defaultApprovals;
-    }
-
     public void setDefaultApprovals(Set<String> defaultApprovals) {
         this.defaultApprovals = defaultApprovals;
     }
 
     public String getCodeApproval() {
+        if (codeApproval == null) {
```

Comment:
would we like to add this in the initialization of the string directly? less raws ;)